generator/internal/codegen_utils.cc

```diff
@@ -164,7 +164,7 @@ std::string ServiceNameToFilePath(absl::string_view service_name) {
 }
 
 std::string ProtoNameToCppName(absl::string_view proto_name) {
-  return "::" + absl::StrReplaceAll(proto_name, {{".", "::"}});
+  return absl::StrReplaceAll(proto_name, {{".", "::"}});
```

Comment:
FWIW, I am not going to review the generated files, I see what this did and trust the tests to do the "Right Thing".